TA-puppet-alert-actions/default/alert_actions.conf

```diff
@@ -1,37 +1,24 @@
-
 [puppet_generate_detailed_report]
 python.version = python3
 description = Generates a puppet:detailed event from a puppet:summary event
 label = Generate detailed Puppet report
 is_custom = 1
 payload_format = json
 icon_path = alert_puppet_generate_detailed_report.png
-param.puppet_enterprise_console = 
-param.puppet_default_user = 
-param.splunk_hec_url = 
-param.splunk_hec_token = 
-param.puppet_action_hec_token = 
-param.puppet_db_url = 
-param.timeout = 
-param.pe_console = 
 
 [puppet_run_task]
 python.version = python3
-description = Run a Bolt Task on the events host with Puppet Enterprise
-label = Run a Bolt Task
+description = Run a Puppet Task on the events host with Puppet Enterprise
```

Comment:
is it important to differentiate here? could we say "run a task..." or would that be too confusing?